packages/security/src/appSecurity/AppSecurity.ts

```diff
@@ -1,16 +1,29 @@
+import axios from "axios";
 import console from "loglevel";
 import {
   ServiceConfiguration,
   ConfigurationService,
-  MetricsService,
-  SecurityMetricsPublisher
- } from "../../../core"; // "@aerogear/core";
+  MetricsBuilder,
+  DefaultMetricsBuilder,
+  isMobileCordova
+ } from "@aerogear/core";
+
+export interface AppSecurityOptions {
+   metricsBuilder?: MetricsBuilder;
+}
 
 export class AppSecurity {
   private static readonly TYPE: string = "security";
   private internalConfig: any;
+  private metricsBuilder: MetricsBuilder;
+
+  constructor(config: ConfigurationService, options?: AppSecurityOptions) {
+    if (!isMobileCordova()) {
+      throw new Error("Cordova Platform Not Detected. This module should not be used.");
```

Comment:
Constructor cannot throw errors. It is very uncommon to handle error for constructor - this will most likely break entire application when running in web.`Console.error` may work here as well.

Reply:
I have no problem removing it , but this module will only work on a device.